src/math/average_mean.jl

```diff
@@ -11,8 +11,8 @@ mean([5, 10, 15, 20, 25, 30, 35])    # returns 20.0
 mean([1, 2, 3, 4, 5, 6, 7, 8])       # returns 4.5
 ```
 
-Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee)
+Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee) and [Rratic](https://github.com/Rratic)
 """
-function mean(nums)
+function mean(nums::Vector)
```

Comment:
I guess this should be better: `mean(nums::Vector)` to `mean(nums::Vector{Number})`

Reply:
tried it. it should be `mean(nums::Vector{T}) where T <: Number`

Reply:
Also, unless there is a conflict with another `mean` method, there is no need to provide types